docs/content/modeling/testing-models.mdx

```diff
@@ -19,6 +19,9 @@ Every <ProductName format={ProductNameFormat.ShortForm}/> model should be tested
 
 The `.fga.yaml` contains tests for <ProductName format={ProductNameFormat.ShortForm}/> authorization models. If you are using Visual Studio Code as your IDE, install the [OpenFGA extension](https://marketplace.visualstudio.com/items?itemName=openfga.openfga-vscode) to enable syntax coloring and validation.
 
+For complete details on the `.fga.yaml` store file format, see [Store File Format](./store-file-format).
```

Comment:
```suggestion
For complete details on the `.fga.yaml` store file format, see [Store File Format](./store-file-format.mdx).
```

Reply:
When referencing direct files here, docusaurus prefers the `.mdx` extension - otherwise it will interpret them as links